src/Analysis/Ast/Test/ImportTests.cs

```diff
@@ -234,5 +238,22 @@ import sys
             var analysis = await GetAnalysisAsync(code);
             analysis.Should().HaveVariable("x").OfType(BuiltinTypeId.Int);
         }
+
+
+        [TestMethod, Priority(0)]
+        public async Task ImportPackageNoInitPy() {
+            var appUri = TestData.GetTestSpecificUri("app.py");
+            await TestData.CreateTestSpecificFileAsync(Path.Combine("top", "sub1", "__init__.py"), string.Empty);
```

Comment:
Is this one of those things that only works in one of Python 2/3? Not sure what happens if you do this in the interpreter off hand.

Reply:
Yes, in 3. Changed per @AlexanderSher suggestions since `GetAllModuleNames` actually did not return packages.

Reply:
We need a parameter, something like `bool includeImplicitPackages`

Reply:
For `GetAllImportable`?

Reply:
Yes

Reply:
Currently there is only one user, what scenario would need to exclude them? I don't mind either way.

Reply:
For Python 2, right?

Reply:
OK.